### PR TITLE
always stringify topics

### LINKED ISSUE #401: Manual publish does not work w/ ecto structs
The example shown here:
https://hexdocs.pm/absinthe/1.4.0-rc.2/Absinthe.Schema.html#subscription/2
(specifically the line `Absinthe.Subscription.publish(pubsub, user, [new_users: user.account_id])`)

doesn't actually work since there is a type mismatch between the subscribed topic and the published one.

Basically, in the config function, `args.id` is a string since the absinthe `ID` type comes in as a string, but since ecto structs get typecasted to numbers and so the `publish/3` call ends up publishing to topics clients are not listening to.

_WORKAROUND_: To get this to work you can either turn your topic key in your publish call into a string to match the topic subscribed like: `Absinthe.Subscription.publish(pubsub, user, [new_users: to_string(user.account_id)]) # Note the to_string at the end here` **OR** convert the topic in the config function to an integer.

Since all phoenix topics are strings, I don't think restricting the topic to strings only is of any major limitation as users will probably expect this is happening under the hood (as I did). Either the docs should be updated to note this, or that key should be converted here: https://github.com/absinthe-graphql/absinthe/blob/master/lib/absinthe/subscription/local.ex#L49 (and above on 46).

### PR DESCRIPTION
**Pull request by benwilson512**
Fixes #401

### CODE REVIEW

File: lib/absinthe/subscription/local.ex [hunk 1]
```diff
@@ -46,7 +46,7 @@ defmodule Absinthe.Subscription.Local do
     do_get_docs(pubsub, field, topic_fun.(mutation_result))
```
Comment (mgwidmann) [inline lib/absinthe/subscription/local.ex]: do you need to `to_string` this or is this for mutation publishes?

Comment (benwilson512) [reply lib/absinthe/subscription/local.ex]: It happens on line 49.

Comment (benwilson512) [reply lib/absinthe/subscription/local.ex]: oh wait no I'm dumb, hold that thought

Comment (benwilson512) [reply lib/absinthe/subscription/local.ex]: fixed via 7b059d0

**Pull request merged**